### PR TITLE
Added ES6 features for JavaScript grammar  #1227

### PR DESCRIPTION
**Pull request by loonydev**
Added import/export statement for Module section. And I think It's close module. https://github.com/antlr/grammars-v4/issues/1227

### CODE REVIEW

File: javascript/JavaScriptParser.g4 [hunk 1]
```diff
@@ -70,6 +72,23 @@ statementList
     : statement+
     ;
 
+importStatement
+    : Import fromBlock
+    ;
+
+
```
Comment (KvanTTT) [inline javascript/JavaScriptParser.g4]: Please remove excess line break.

Comment (loonydev) [reply javascript/JavaScriptParser.g4]: I'm working yet

Comment (loonydev) [reply javascript/JavaScriptParser.g4]: Done

**Comment by loonydev** [comment]
I think it's final version for Module

**Comment by KvanTTT** [comment]
That's why I suggets replacing ~~`Modules`~~ with `Modules` in README :)

**Comment by loonydev** [comment]
Yep. Replaced in the last force push. 
Sorry for a lot of FP, I just added import, but can't stop, while not fix all in `Module`

**Comment by loonydev** [comment]
Don't merge, generators coming)

**Comment by loonydev** [comment]
Okay, looks good. Let's review.

**Comment by KvanTTT** [comment]
Very good :+1: Thanks a lot.

**Pull request merged**